deliveries/php-scale-lib_milestone_2.md

```diff
@@ -0,0 +1,17 @@
+# Milestone Delivery :mailbox:
+
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**
+
+* **PR Link:** [php-scale-lib](https://github.com/w3f/Open-Grants-Program/pull/235)
+* **Milestone Number:** 2
+
+Please provide a list of all deliverables of the milestone extracted from the initial application and a link to the deliverable itself. Ideally all links inside the below table should include a commit hash, which should be used for testing.
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 1. | Source code   | https://github.com/gmajor-encrypt/php-scale-codec |
+| 2. | License       | MIT license |
+| 3. | Unit Tests    | https://github.com/gmajor-encrypt/php-scale-codec/tree/master/test/Codec | make test |
+| 4. | Documentation | https://github.com/gmajor-encrypt/php-scale-codec#custom-types |
+| 5. | Packagist     | https://packagist.org/packages/gmajor/substrate-codec-php |
```

Comment:
Could you please structure the list of deliverables as in your [application](https://github.com/w3f/Grants-Program/blob/master/applications/php-scale-lib.md#milestone-2-example--additional-features) and provide a link to where each of them is implemented?